tests/unit/providers/vector_io/conftest.py

```diff
@@ -94,7 +96,7 @@ def sample_embeddings_with_metadata(sample_chunks_with_metadata):
     return np.array([np.random.rand(EMBEDDING_DIMENSION).astype(np.float32) for _ in sample_chunks_with_metadata])
 
 
-@pytest.fixture(params=["milvus", "sqlite_vec", "faiss"])
+@pytest.fixture(params=["milvus", "sqlite_vec", "faiss", "chroma", "qdrant"])
```

Comment:
@franciscojavierarceo I added "chroma" in here - IIRC you were mentioning about this fix today morning.